gulpfile.babel.js

```diff
@@ -0,0 +1,264 @@
+import gulp from 'gulp';
+import less from 'gulp-less';
+import cleanCss from 'gulp-clean-css';
+import sourcemaps from 'gulp-sourcemaps';
+import Autoprefix from 'less-plugin-autoprefix';
+import eslint from 'gulp-eslint';
+import imagemin from 'gulp-imagemin';
+import rollup from 'gulp-rollup';
+import uglify from 'gulp-uglify';
+import babel from 'gulp-babel';
+import rename from 'gulp-rename';
+import del from 'del';
+import nsp from 'gulp-nsp';
+import * as path from 'path';
+import changed from 'gulp-changed';
+import concat from 'gulp-concat';
+import browserSync from 'browser-sync';
+import browserify from 'browserify';
+import browserifyInc from 'browserify-incremental';
+import source from 'vinyl-source-stream';
+import buffer from 'vinyl-buffer';
+import notify from 'gulp-notify';
+import plumber from 'gulp-plumber';
+import processhtml from 'gulp-processhtml';
+import uncss from 'gulp-uncss';
+
+const isProdBuild = () => process.argv.filter(val => val.toLowerCase().indexOf('-prod') !== -1).length > 0;
+const pkgJson = require('./package.json');
+const server = browserSync.create();
+const mainDirectories = {
+	dev: './server/',
+	dist: './dist/'
+};
+
+const settings = {
```

Comment:
see 3433dca. please let me know if I should include other things like e.g. `onError`